tests/lib/rules/camelcase.js

```diff
@@ -79,6 +79,30 @@ ruleTester.run("camelcase", rule, {
         {
             code: "import { no_camelcased as camelCased, anoterCamelCased } from \"external-module\";",
             parserOptions: { ecmaVersion: 6, sourceType: "module" }
+        },
+        {
+            code: "const { no_camelcased } = bar;",
```

Comment:
I think we're missing a test case for 

```

Reply:
I think that's covered with `var { category_id: category } = query;`